fluent.runtime/fluent/runtime/fallback.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+
+import codecs
+import os
+import six
+
+
+class FluentLocalization(object):
+    """
+    Generic API for Fluent applications.
+
+    This handles language fallback, bundle creation and string localization.
+    It uses the given resource loader to load and parse Fluent data.
+    """
+    def __init__(
+        self, locales, resource_ids, resource_loader,
+        use_isolating=False,
+        bundle_class=None, functions=None,
+    ):
```

Comment:
Is this PR the right place to comment on the design of the `FluentLocalization` API? Would you prefer to move this discussion to Discourse perhaps?

I'd like to suggest using the pattern we use in `@fluent/react` and `@fluent/dom`, i.e. rather than pass a lot of config parameters to the constructor, we pass an iterator over bundles created by the user. This allow the user of the API to be in charge of the exact bundle class used, the options and functions used to initialize it, and the loading logic.